### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
Found via `codespell -L crate,isnt`.

### CODE REVIEW

File: moreexamples.md [hunk 1]
```diff
@@ -318,7 +318,7 @@ WantedBy=sockets.target
 ```
 
 
-`/etc/systemd/system/qqq.service` (note the absense of  `@`).
+`/etc/systemd/system/qqq.service` (note the absence of  `@`).
```
Comment (vi) [inline moreexamples.md:321]: What is the reason for this change?

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:
